plugins/org.jkiss.dbeaver.ext.mssql/src/org/jkiss/dbeaver/ext/mssql/edit/SQLServerIndexManager.java

```diff
@@ -143,4 +144,11 @@ protected String getDropIndexPattern(SQLServerTableIndex index)
         return "DROP INDEX " + index.getName() + " ON " + index.getTable().getFullyQualifiedName(DBPEvaluationContext.DDL);
     }
 
+
+    @Override
+    protected void addObjectModifyActions(DBRProgressMonitor monitor, DBCExecutionContext executionContext, List<DBEPersistAction> actionList, ObjectChangeCommand command, Map<String, Object> options) throws DBException {
```

Comment:
Please add annotation (IDEA should suggest this automatically) + format parameters on new lines